packages/core/src/Constants/contract.js

```diff
@@ -222,6 +190,38 @@ export const getSupportedContracts = is_high_low => ({
         name: localize('No Touch'),
         position: 'bottom',
     },
+    RUNHIGH: {
+        name: localize('Only Ups'),
+        position: 'top',
+    },
+    RUNLOW: {
+        name: localize('Only Downs'),
+        position: 'bottom',
+    },
+    EXPIRYMISS: {
+        name: localize('Ends Outside'),
+        position: 'top',
+    },
+    EXPIRYRANGE: {
+        name: localize('Ends Between'),
+        position: 'bottom',
+    },
+    RANGE: {
+        name: localize('Stays Between'),
+        position: 'top',
+    },
+    UPORDOWN: {
+        name: localize('Goes Outside'),
+        position: 'bottom',
+    },
```

Comment:
Are you moving the rest of the contracts into this list for the future supported contracts?